### PR TITLE
Added an API for getting nodes inside a group

### PR DESCRIPTION
**Pull request by remygrandry**
I know the API name doesn't respect our naming conventions, but it is weird to have a single lowercase API among the others

### CODE REVIEW

File: imgui_node_editor.cpp [hunk 1]
```diff
@@ -1195,6 +1195,17 @@ void ed::EditorContext::Begin(const char* id, const ImVec2& size)
     m_LastSelectedObjects = m_SelectedObjects;
 }
 
+void ed::EditorContext::GetGroupContainedIds(NodeId id, std::vector<NodeId>& ids)
```
Comment (hendrik-skubch) [inline imgui_node_editor.cpp]:
```suggestion
void ed::EditorContext::GetGroupContainedIds(NodeId id, std::vector<NodeId>* out_ids)
```

File: imgui_node_editor.cpp [hunk 2]
```diff
@@ -1195,6 +1195,17 @@ void ed::EditorContext::Begin(const char* id, const ImVec2& size)
...
+{
+    ids.clear();
+    auto* group = GetNode(id);
+    for (auto node : m_Nodes) {
```
Comment (hendrik-skubch) [inline imgui_node_editor.cpp]:
```suggestion
    for (const auto& node : m_Nodes) {
```

File: imgui_node_editor.cpp [hunk 3]
```diff
@@ -1195,6 +1195,17 @@ void ed::EditorContext::Begin(const char* id, const ImVec2& size)
     m_LastSelectedObjects = m_SelectedObjects;
 }
 
+void ed::EditorContext::GetGroupContainedIds(NodeId id, std::vector<NodeId>& ids)
+{
+    ids.clear();
+    auto* group = GetNode(id);
```
Comment (hendrik-skubch) [inline imgui_node_editor.cpp]:
```suggestion
    const auto* group = GetNode(id);
```

File: imgui_node_editor_api.cpp [hunk 4]
```diff
@@ -176,6 +176,11 @@ ImVec2 ax::NodeEditor::GetGroupMax()
     return s_Editor->GetHintBuilder().GetGroupMax();
 }
 
+void ax::NodeEditor::GetGroupContainedIds(NodeId id, std::vector<NodeId>& ids)
```
Comment (hendrik-skubch) [inline imgui_node_editor_api.cpp]:
```suggestion
void ax::NodeEditor::GetGroupContainedIds(NodeId id, std::vector<NodeId>* out_ids)
```

**Review by hendrik-skubch — COMMENTED** [review]
That's fine, staying consistent with the lib you are working in is more important.
Just a few comments regarding constness and reference arguments.

**Comment by remygrandry** [comment]
> That's fine, staying consistent with the lib you are working in is more important. Just a few comments regarding constness and reference arguments.

Oops, I think I looked to much the code around while coding those. Thanks !

**Review by hendrik-skubch — APPROVED** [review]
lgtm, thanks!

**Pull request merged**